cmd/commands/service.go

```diff
@@ -304,6 +331,33 @@ Additional curl arguments and flags may be specified after a double dash (--).`,
 	return command
 }
 
+// Print any HTTP errors in the given string to the given writer
+func PrintCurlHttpErrors(curlErrOutput string, w io.Writer) {
+	matched, err := regexp.MatchString("< HTTP/[\\d\\.]* 200", curlErrOutput)
```

Comment:
you can instead use `regexp.MustCompile` that gives you a Regexp struct or panics, and then call `MatchString` without the extraneous `err` (same applies below)